solutions/factorial_calculator.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
```

Comment:
@linahKhayri Great work! Well structured and clear code. I like number of unit tests.